src/content/docs/mobile-monitoring/new-relic-mobile-maui-dotnet/monitor-your-net-maui-application.mdx

```diff
@@ -0,0 +1,543 @@
+---
+title: Monitor your .NET MAUI app
+tags:
+    - .NET MAUI
+    - Mobile monitoring
+    - Hybrid framework
+    - Get started
+metaDescription: "New Relic's .NET MAUI agent for Android and iOS: Features, compatibility, requirements, installation, and upgrade procedures."
+---
+
+import mobileFlutterSummary from 'images/mobile_screenshot-full_flutter-summary.webp'
+
+Monitor your .NET MAUI application with our New Relic .NET MAUI agent. Capture C# errors, track specific network requests, use distributed tracing to drill down into handled exceptions, and create custom tracking events and metrics to fully capture your end users' experience.
+
+<img
+  title="Flutter summary view in the UI"
+  alt="Summary view of a flutter app in New Relic"
+  src={mobileFlutterSummary}
+  style={{ maxWidth: '70%' }}
+/>
+
+<figcaption>
+  **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Mobile > (select an app) > Summary**: View .NET MAUI data, track HTTP requests and errors, and monitor how your app is performing over time.
+</figcaption>
+
+Ready to get started? Follow the steps below! 
+
+## Install the .NET MAUI SDK
+<Steps>
+
+<Step>
+### Review the requirements
+
+Before you install the .NET MAUI agent, make sure your .NET MAUI app meets these version requirements:
+
+For Android-native apps:
+    * Android 7 (API 24) or higher
+
+For iOS-native apps:
+    * iOS 11 or higher
+    * Latest release of Xcode
+</Step>
+
+<Step>
+### Add the Flutter agent to your project
+
+First, you'll need to add the agent package `NewRelic.MAUI.Plugin` to your MAUI project. In your `DIRECTORY`, add the following to `FILE_NAME`: 
```

Comment:
Where should they add `NewRelic.MAUI.Plugin` to specifically in their MAUI project?